src/test/java/net/minecraftforge/debug/misc/GrindstoneEventTest.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.debug.misc;
+
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.Items;
+import net.minecraftforge.event.GrindstoneEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+
+@Mod("grindstone_event_test")
+@Mod.EventBusSubscriber
+public class GrindstoneEventTest {
```

Comment:
Please add an `ENABLED` field, defaulted to `false`, which disables the event listeners (manually add the listeners using `MinecraftForge.EVENT_BUS#addListener`/`MinecraftForge.EVENT_BUS#register`)